x-pack/platform/plugins/shared/cases/public/components/system_actions/cases/cases_params.tsx

```diff
@@ -133,6 +140,14 @@ export const CasesParamsFieldsComponent: React.FunctionComponent<
         return;
       }
 
+      const timeSize = key === 'timeWindowSize' ? value : timeWindowSize;
+      const timeUnit = key === 'timeWindowUnit' ? value : timeWindowUnit;
+      const numericTimeSize = parseInt(timeSize, 10);
+
+      const showWindowWarning = timeUnit === 'm' && numericTimeSize >= 5 && numericTimeSize <= 20;
+
+      setShowTimeWindowWarning(showWindowWarning);
```

Comment:
```suggestion
      const timeSize = key === 'timeWindowSize' ? value : timeWindowSize;
      const timeUnit = key === 'timeWindowUnit' ? value : timeWindowUnit;
      const numericTimeSize = parseInt(timeSize, 10);

      const showWindowWarning = timeUnit === 'm' && numericTimeSize >= 5 && numericTimeSize <= 20;

      setShowTimeWindowWarning(showWindowWarning);
```

Can't we use `showWarning` from above?